### ISSUE TITLE
Text is copied into the image section 

### ISSUE THREAD

**Issue by stevenveenma** [TheLastBen/fast-stable-diffusion#1997]
In situations where I upload images, f.i. img2img and controlnet,  I run into trouble when I copy text to the prompt. The text than overrides the image although I don't paste it there. When I run the model on this by accident, I run into errors.

**Comment by TheLastBen** [comment]
any screenshot of the process and how to reproduce the error ?

**Comment by stevenveenma** [comment]
Ok, I tested a few things and discovered the problem is limited to the use of Onenote. I use Onenote to store notes and prompts. I tested two versions of Onenote on a Windows 10 system: Onenote 2010 (quite obsolete) and Onenote for Windows 10. Both have the same behaviour. Pictures below. I also tested a notepad textfile en MS Word, but not problems there. 

![Starting point](https://user-images.githubusercontent.com/28566657/232585603-c3f84d80-1005-40ab-a782-fee5c0f8f587.png)

![Test onenote Windows 10](https://user-images.githubusercontent.com/28566657/232585630-057286a3-a7c1-4913-b80f-c485b82f2cc4.png)

![Test onenote Windows 10](https://user-images.githubusercontent.com/28566657/232585664-a641af41-e8c3-4205-a338-526412d596c9.png)
